jekyll/_cci2/vs-code-extension-quickstart.adoc

```diff
@@ -0,0 +1,117 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+---
+= Get started with the VS Code extension
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A quickstart tutorial for the CircleCI VS Code extension.
+:icons: font
+:toc: macro
+:toc-title:
+
+This tutorial is a quickstart guide to setting up and using the official CircleCI VS Code extension. You can read more about the extension's capabilities in the xref:vs-code-extension-overview#[overview]. 
+
+[#prerequisits]
+== Prerequisites
+
+Before starting this tutorial, make sure you have:
+
+* Installed the Visual Studio Code editor
++
+NOTE: The CircleCI extension is compatible with VS Code version `1.67` or above.
+
+* A CircleCI account (you can link:https://circleci.com/signup/[sign up for free])
+
+* A xref:create-project#[project] you follow that is building on CircleCI
+
+* CircleCI server users: your self-hosted server URL
+
+[#install-the-extension]
+== 1. Install the extension
+
+There are two ways to install the CircleCI extension:
+
+* In VS Code, click the **Extensions** icon on the left-hand Activity Bar, or open **View > Extensions**.
+
+* Or, download the extension from the link:https://marketplace.visualstudio.com/items?itemName=circleci.circleci[VS Code Marketplace].
+
+[#authenticate-into-circleci]
+== 2. Authenticate into CircleCI
+
+. In VS Code, open the CircleCI panel by clicking on the CircleCI icon on the activity bar, then click “Log in”.
+
+. You will be prompted to enter a xref:managing-api-tokens#overview[personal API token]. You may create one in your CircleCI link:https://app.circleci.com/settings/user/tokens[**User Settings > Personal API Tokens**.]
++
+**CircleCI server**: Make sure to check the "I'm a CircleCI Server customer," then enter your self-hosted server URL.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_login.png[Log in to CircleCI from VS Code]
+
+[#set-up-your-project-in-vs-code]
+== 3. Set up your project in VS Code
+
+NOTE: The pipelines manager does not yet support GitLab projects.
```

Comment:
Just wondering, does this note to GitLab users actually apply to the following section? And are "pipelines panel" and "pipelines manager" used interchangeably? 

Also wondering, if this view isn't available for GitLab, what do they see instead? Should we suggest where in the quickstart they should jump to?

Reply:
@rosieyohannan I think they have been used somewhat interchangeably although I think I'd refer to the specific portion of the VS Code UI as "panel" and then say "pipelines manager" for the entire feature. @BeFunes thoughts?

Reply:
Yes correct. The Panel is the 'physical' panel, the manager is the functionality. 

For Gitlab, projects aren't detected automatically nor do they appear in the dropdown. 
If a user has a mix of Gitlab + Github projects, then they can use the Pipelines Manager on the Github ones, otherwise they can only use the Config Helper - which would be point 7 in this guide